### ISSUE TITLE
Possible dead lock with 0RTT and depleted congestion window

### ISSUE THREAD

**Issue by tatsuhiro-t** [quicwg/base-drafts#2984]
Lets suppose that client sends HTTP/3 request in 0RTT, and server sends its response in Short packets and it consumes all available congestion window.
And somehow server Initial and/or Handshake are lost.  Client cannot send ACK for Short packet because it does not have 1RTT key.  Server cannot resend Initial/Handshake packet because congestion window is 0.
I think that I have to make Short packets lost and reclaim in_flight bytes.  But I'm not sure https://tools.ietf.org/html/draft-ietf-quic-recovery-22#appendix-A.9 covers this case.
It looks to me that unless pn_space becomes 1rtt pn space, no processing done for Short packets.

**Comment by marten-seemann** [comment]
If I understand correctly, the retransmissions would be sent when the PTO timer fires in this case, and therefore are probe packets. As such, they're not blocked by the congestion controller.

**Comment by tatsuhiro-t** [comment]
According to the pseudo code in A-9, PTO does not fire if endpoint has unacknowledged crypto data.

**Comment by ianswett** [comment]
In the editor's draft, PTO replaces the handshake timer, TLP and RTO.  In the current editor's draft, it's armed until the client has 1-RTT keys.  So the client must retransmit something in Initial or Handshake encryption levels to move the handshake along.

Given that, I'm unclear on what the issue is?

**Comment by tatsuhiro-t** [comment]
Thank you for replies.  The issue is against draft-22, and I haven't checked editor's copy.
At this point, I realized that this issue is a duplicate of #2779.  .5RTT response just accelerates the issue.  The root cause is that we cannot declare handshake packets lost in the draft-22.  If it is fixed in editor's copy then you can close #2779 as well.
